pandas/tests/scalar/timestamp/test_constructors.py

```diff
@@ -382,7 +399,8 @@ def test_bounds_with_different_units(self):
         for date_string in out_of_bounds_dates:
             for unit in time_units:
                 dt64 = np.datetime64(date_string, unit)
-                with pytest.raises(ValueError):
+                msg = "Out of bounds"
+                with pytest.raises(ValueError, match=msg):
```

Comment:
is this OutOfBoundsDatetime

Reply:
yes it is, sorry i just didn't pay attention to the old code